catala/somrius/somrius.md

```diff
@@ -0,0 +1,9 @@
+Somrius. Content i feliç com quan eres un nen.
+
+El somni de la teva vida de trobar una casa de núvols de sucre, com el d'en Hansel i la Gretel, s'ha complert!
+
+Prens un bocí de núvol i recordes que vols volar. unes ales t'apareixen a l'esquena. Ara tens un dilema, què fas?
+
+[Continues menjant núvols de sucre](create-your-own-adventure/catala/menjar/menjar.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.